src/functionalTest/resources/scenarios/RIA-706-upload-additional-evidence-submit.json

```diff
@@ -60,7 +60,7 @@
     "caseData": {
       "template": "minimal-appeal-submitted.json",
       "replacements": {
-        "additionalDocuments": [
+        "additionalEvidenceDocuments": [
```

Comment:
did we change an existing field name here?  Why was that?  Just wondering if it affects any existing functionality.

Reply:
it seemed to have been changed in the spreadsheet code.  This test was failing.